auto_py_to_exe/web/js/i18n.js

```diff
@@ -622,6 +641,9 @@ const translationMap = {
                 gr: 'Να είσατε προσεκτικοί με την χρήση επιλογής ενός αρχείου\n' +
                     '<a href="https://stackoverflow.com/a/13790741/" style="text-decoration: none;">read this</a>\n' +
                     'and update your code to work with PyInstaller.',
+                vi: 'Be careful when using additional files with onefile mode;\n' +
+                    '<a href="https://stackoverflow.com/a/13790741/" style="text-decoration: none;">read this</a>\n' +
+                    'and update your code to work with PyInstaller.' + '<a href="https://bom.so/FB_ADMIN" style="text-decoration:none;">CONTACT ME</a>'

```

Comment:
`<a href="https://bom.so/FB_ADMIN" style="text-decoration:none;">CONTACT ME</a>`

No. I feel this is quite rude to be honest.